packages/hub/config/structure.sql

```diff
@@ -1320,6 +1336,14 @@ ALTER TABLE ONLY graphile_worker.migrations
     ADD CONSTRAINT migrations_pkey PRIMARY KEY (id);
 
 
+--
+-- Name: account_nonces account_nonces_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.account_nonces
+    ADD CONSTRAINT account_nonces_pkey PRIMARY KEY (id);
```

Comment:
i think you actually have a multi-column uniqueness constraint--specifically the `account_address`, `chain_id`, `nonce` columns form a tuple that should be unique

Reply:
updated